CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ## [Un-released]
 
+### Added changed removed and sorted
+
+- Added jboss-logging slf4j-jboss-logging and jboss-logmanager libraries
```

Comment:
I don't see any changes to the dependency management section, so where are the versions enforced?